lua/opencode/config.lua

```diff
@@ -163,6 +167,45 @@ M.defaults = {
 
 M.values = vim.deepcopy(M.defaults)
 
+--- Check if a keymap configuration uses the old format (internal use only)
+--- @param keymap_config table
+--- @return boolean
+local function is_old_format(keymap_config)
```

Comment:
With the updated config format you could just check for the `window` or `global` key under keymaps